src/components/dev-dot-content/mdx-components/mdx-alert/docs.mdx

```diff
@@ -68,33 +68,72 @@ The `MdxInlineAlert` component is used in long-form MDX content to highlight var
 	soufflé dessert chocolate gingerbread pie.
 </MdxInlineAlert>
 
-## Examples
+## Authoring in MDX
```

Comment:
Link to these docs in tutorials repo here https://github.com/hashicorp/tutorials/pull/1050